samples/drivers/watchdog/src/main.c

```diff
@@ -10,9 +10,14 @@
 #include <watchdog.h>
 #include <misc/printk.h>
 
-#define WDT_DEV_NAME   CONFIG_WDT_0_NAME
 #define WDT_FEED_TRIES 5
 
+#ifndef DT_WDT_0_NAME
```

Comment:
Just a detail. This inversed logic may be confusing. I think it is better to use "if this symbol is defined, use it; otherwise, use the default or fallback one". So in this case I'd use `#ifdef CONFIG_WDT_0_NAME`.

Reply:
Changed